app/models/orchestration.rb

```diff
@@ -35,12 +35,12 @@ def run
   # rubocop:enable Rails/SkipsModelValidations,Metrics/CyclomaticComplexity
 
   def self.run(kind: :bootstrap, params: nil)
-    raise OrchestrationOngoing unless runnable?
+    raise OrchestrationOngoing if runnable?
```

Comment:
to be correct i would keep this `raise OrchestrationOngoing unless runnable?`

otherwise it read as raise Error when runnable

Reply:
ok